### PR TITLE
Fixed several data models

### PR DESCRIPTION
**Pull request by devel0peter**

### CODE REVIEW

File: Meraki.Api/Data/Tag.cs [hunk 1]
```diff
@@ -18,5 +18,5 @@ public class Tag
 	/// </summary>
 	/// <value>The privilege of the dashboard administrator on the tag</value>
 	[DataMember(Name = "access")]
-	public string Access { get; set; } = string.Empty;
+	public AccessPrivilege Access { get; set; }
```
Comment (danielabbatt) [inline Meraki.Api/Data/Tag.cs:21]: Need ApiAccess attribute

**Review by danielabbatt — COMMENTED** [review]
Possibly missing an attribute, rest looks ok I think.

File: Meraki.Api/Data/Tag.cs (hunk 1 above)
Comment (devel0peter) [reply Meraki.Api/Data/Tag.cs:21]: Added attribute.

**Review by danielabbatt — APPROVED** [review]
Looks good

**Pull request merged**